src/components/AssetSearch/components/AssetList.tsx

```diff
@@ -25,6 +25,8 @@ export type AssetData = {
   showPrice?: boolean
   onImportClick?: (asset: Asset) => void
   showRelatedAssets?: boolean
+  useWindowScroll?: boolean
```

Comment:
🤮 but hey, that's from `react-virtuoso` types and we just honor that

<img width="630" height="93" alt="image" src="https://github.com/user-attachments/assets/fd3b4913-fe3d-41d7-bdf1-37eed6f0ddf1" />